src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+  const body = document.querySelector('body');
+
+  div.className = `notification ${type}`;
+
+  div.insertAdjacentHTML("afterbegin",
```

Comment:
```suggestion
  div.insertAdjacentHTML('afterbegin',
```
In JS strings must use singlequote